app/src/main/java/org/jellyfin/androidtv/di/PreferenceModule.kt

```diff
@@ -10,6 +11,9 @@ val preferenceModule = module {
 	single { LiveTvPreferences(get(userApiClient)) }
 	single { UserSettingPreferences(get(userApiClient)) }
 	single { AuthenticationPreferences(androidApplication()) }
-	single { UserPreferences(androidApplication()) }
+	single { UserPreferences(androidApplication()) } binds arrayOf(
+		UserPreferences::class,
+		PreferenceStore::class
+	)
```

Comment:
No need to bind to the UserPreferences class as the app should never want to retrieve a store by this interface as it doesn't contain any preferences